docs/dagster-university/pages/dagster-dbt/lesson-3/3-representing-the-dbt-project-in-dagster.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Lesson 3: Representing the dbt project in Dagster'
+module: 'dagster_dbt'
+lesson: '3'
+---
+
+# Representing the dbt project in Dagster
+
+As you’ll frequently point your Dagster code to the `target/manifest.json` file and your dbt project in this course, it’ll be helpful to keep a reusable representation of the dbt project. This can be easily done using the `DbtProject` class.
+
+Create a new file in the `assets` directory called `dbt.py`. Open that file and add the following import at the top:
+
+```python
+from pathlib import Path
+
+from dagster_dbt import DbtProject
+```
+
+The `Path` class from the `pathlib` standard library will help us create an accurate pointer to where our dbt project is. The `DbtProject` class is imported from the `dagster_dbt` package that we installed earlier. After the import, add the following code: 
+
+```python
+dbt_project = DbtProject(
+  project_dir=Path(__file__).joinpath("..", "..", "..", "analytics").resolve(),
```

Comment:
I know we did this in the previous lesson as well, but I'm not a fan of repeating `..` relative paths.

Would it be more intuitive if we used dagster home, or `DagsterInstance.get().root_directory()`? This can merge before this change, but wanted to bring it up.

Reply:
I understand your point, but I think we should avoid this for two main reasons:
- it is common to see a dbt project outside the Dagster project directory - we would need to resolve the path using `DagsterInstance.get().root_directory()` and more `..` to indicate the dbt project location.
- the current scaffold template uses `..` relative paths. If we want to change that in Dagster University, we should start by the scaffold.